fuzz.yaml

```diff
@@ -0,0 +1,32 @@
+base: ubuntu:16.04
+language: c
+setup:
+- sudo bash -c "echo 'deb-src http://archive.ubuntu.com/ubuntu/ xenial main restricted universe multiverse' >>/etc/apt/sources.list"
+- sudo apt-get update -y
+- sudo apt-get build-dep -y systemd
+- sudo apt-get install -y python3-pip
+- pip3 install meson ninja
+- export PATH="$HOME/.local/bin/:$PATH"
+- CC=$FUZZ_CC CXX=$FUZZ_CXX meson -Dfuzzbuzz=true -Dfuzzbuzz-engine-dir=$(dirname "$FUZZ_ENGINE") -Dfuzzbuzz-engine=$(cut -d. -f1 <(basename "$FUZZ_ENGINE")) -Db_lundef=false ./build
+- ninja -v -C ./build fuzzers
+environment:
+targets:
+- name: fuzz-compress
+  harness:
+    binary: ./build/fuzz-compress
+- name: fuzz-unit-file
+  harness:
+    binary: ./build/fuzz-unit-file
+  corpus: ./test/fuzz/fuzz-unit-file
+- name: fuzz-journald-syslog
+  harness:
+    binary: ./build/fuzz-journald-syslog
+  corpus: ./test/fuzz/fuzz-journald-syslog
+- name: fuzz-netdev-parser
+  harness:
+    binary: ./build/fuzz-netdev-parser
+  corpus: ./test/fuzz/fuzz-netdev-parser
+- name: fuzz-network-parser
+  harness:
+    binary: ./build/fuzz-network-parser
+  corpus: ./test/fuzz/fuzz-network-parser
```

Comment:
I wonder if we could generate this part automatically. So far we avoided explicit configuration for each fuzzer.

Reply:
I've been looking at `tools/oss-fuzz.sh`, which allows us to kind of avoid explicit configuration, and I'm afraid I haven't been able to come up with a generator that would be general enough yet . At this point some fuzzers don't work on FuzzBuzz (which makes it a bit harder to generate everything automatically) but I think when all the fuzzers are ready to be run there, it should be doable.

Reply:
Right. This explicit form is good enough for now.

Reply:
We've been thinking about nice ways to have Fuzzbuzz automatically pick up targets - iirc oss-fuzz automatically detects Makefile targets with certain names (or something similar). Let me know if you have any ideas/thoughts on this too

Reply:
As far as I know, OSS-Fuzz can't automagically discover fuzz targets either. It considers every binary put in a special directory a fuzzer and dictionaries, corpora and everything else have to be placed there too and named accordingly, that is there're conventions everybody should follow. It almost always work and when it doesn't the `.options` files have to be added to, for example, pass additional options to asan (which didn't work when the engine was AFL last time I checked :-)). So I'm totally fine with the explicit configuration :-)